src/WebJobs.Script.WebHost/Standby/IStandbyManager.cs

```diff
@@ -9,8 +9,6 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost
 {
     public interface IStandbyManager
     {
-        IChangeToken GetChangeToken();
```

Comment:
I noticed this wasn't actually used anywhere, so pulled it out.

Reply:
Hmm, I don't even know what that was ever used for

Reply:
It was originally for the refresh token when we go into standby mode (which signals that settings need to be refreshed). @fabiocav moved this out to its own class (`StandbyChangeTokenSource`) and just missed the cleanup of removing these properties, I believe.